src/main/java/com/fasterxml/jackson/databind/util/EnumValues.java

```diff
@@ -40,7 +41,7 @@ public static EnumValues construct(SerializationConfig config, Class<Enum<?>> en
         return constructFromName(config, enumClass);
     }
 
-    public static EnumValues constructFromName(MapperConfig<?> config, Class<Enum<?>> enumClass)
+    public static EnumValues constructFromName(SerializationConfig config, Class<Enum<?>> enumClass)
     {
```

Comment:
technically an API change (causing backward compatibility issues). Can you keep `public static EnumValues constructFromName(MapperConfig<?> config, Class<Enum<?>> enumClass)` and deprecate it - you can add the new method too

Reply:
Good catch -- see my other note, I changed things so that `DatatypeFeature` is now accessible via `MapperConfig`, not just its sub-classes.